README.md

```diff
@@ -1 +1,120 @@
-# nva-cristin-institutions
\ No newline at end of file
+# nva-cristin-institutions
```

Comment:
That is a fine ReadMe.

(Maybe something about how to build or test locally might be nice for later use)